capsules/src/test/random_alarm.rs

```diff
@@ -19,7 +19,7 @@ impl<'a, A: Alarm<'a>> TestRandomAlarm<'a, A> {
         TestRandomAlarm {
             alarm: alarm,
             counter: Cell::new(value),
-            expected: Cell::new(A::ticks_from_seconds(0)),
+            expected: Cell::new(0u32.into()),
```

Comment:
We could change it to `alarm.ticks_from_seconds(0)`, however I do think that the assumption that 0 is the same will always hold true since tick is just another scaled measure of time, and 0 * any_scale => 0

Reply:
I don't feel so strongly about this. Personally, I'd probably write `self.alarm.ticks_from_seconds(0)` to reinforce what the number signifies, but your logic makes sense too.

Unless anyone else does feel strongly and wants to follow up, this can probably stay as-is.

Reply:
I agree with @brghena. I agree that @jettr's logic is correct, however `self.alarm.ticks_from_seconds(0)` makes it much more obvious that we're talking about some `Ticks` here. The `Into` trait is good for when it's obvious what type the conversion will result in, but arguable here it's not the case. I'd also be okay with `Ticks::from(0u32)`.